plugins/@grouparoo/pipedrive/src/lib/connect.ts

```diff
@@ -0,0 +1,13 @@
+import { SimpleAppOptions } from "@grouparoo/core";
+import pipedrive from "pipedrive";
+import { EnhancedPersonsController } from "./client";
+
+export async function connect(appOptions: SimpleAppOptions) {
+  pipedrive.Configuration.apiToken = appOptions.apiToken;
+
+  pipedrive.EnhancedPersonsController = new EnhancedPersonsController(
+    pipedrive
+  );
```

Comment:
This may be a bug.  The apiToken looks to be set globally on `import pipedrive from "pipedrive"`... which means that if there are 2 Pipedrive Apps the wrong api key could be used.  This should be an instance (`new pipedrive()`), or we need to duplicate the object per `connect`...

Reply:
Good catch! Yeah, the pipedrive library uses a singleton pattern. Hadn't considered the possibility of having two different apps. I tried cloning the pipedrive object in multiple ways, but it doesn't work since internally it works under the assumption that it is a singleton. The library didn't do much over the standard REST APIs so I implemented our own client.